ssh/tailssh/tailssh.go

```diff
@@ -431,7 +439,18 @@ func (srv *server) handleSSH(s ssh.Session) {
 // Any action with a Message in the chain will be printed to ss.
```

Comment:
docs on line 436 (new side 436) are stale now. "returns action" but no "action" is passed in now.